src/omotes_sdk/internal/common/broker_interface.py

```diff
@@ -348,46 +393,20 @@ def remove_queue_subscription(self, queue_name: str) -> None:
             self._remove_queue_subscription(queue_name), self._loop
         ).result()
 
-    def receive_next_message(
```

Comment:
Removed as we merged `QueueSubscriptionConsumer` and `QueueSingleMessageConsumer`. The single messages functionality is now maintained by setting `delete_after_messages=1`